Content.Shared/Humanoid/HumanoidVisualLayersExtension.cs

```diff
@@ -82,9 +82,15 @@ public static IEnumerable<HumanoidVisualLayers> Sublayers(HumanoidVisualLayers l
             }
         }
 
+        // Begin DeltaV - We want this data w/o needing a BodyPartComponnet
         public static HumanoidVisualLayers? ToHumanoidLayers(this BodyPartComponent part)
         {
-            switch (part.PartType)
+            return HumanoidVisualLayersExtension.ToHumanoidLayers(part.PartType, part.Symmetry);
+        }
+
+        public static HumanoidVisualLayers? ToHumanoidLayers(BodyPartType partType, BodyPartSymmetry partSymmetry)
```

Comment:
put this before BodyPartType so its actually an extension method
update the use of it accordingly